### PR TITLE
Changes for Urdu Language

### PR DESCRIPTION
**Pull request by navneetsn18**
Related to #25

@eldadfux @spielers  I have translated appwrite locale and email templates in Urdu.

Please review the changes and let me know if any fixes required.

### CODE REVIEW

**Comment by spielers** [comment]
Hey, @navneetsn18 I suggest not editing the init.php file. I think that's causing a Build failure.

**Comment by navneetsn18** [comment]
Okk @spielers I will try to fix the issue..

**Comment by navneetsn18** [comment]
@spielers Now is it ok..

**Comment by spielers** [comment]
@navneetsn18 Thanks now it's Fine.

File: app/config/locale/ur.countries.php [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    'BT' => 'بھوٹان',
+    'BW' => 'بوٹسوانا',
+    'CF' => 'مرکزی افریقی جمہوریت',
+    'CA' => 'کینیڈا',
+    'CH' => 'سوئٹزرلینڈ',
+    'CL' => 'چلی',
+    'CN' => 'چین',
+    'CI' => 'آئیوری کوسٹ',
+    'CM' => 'کیمرون',
+    'CD' => 'DR کانگو',
```
Comment (spielers) [inline app/config/locale/ur.countries.php:39]: retranslate this

**Review by spielers — CHANGES_REQUESTED** [review]

**Comment by eldadfux** [comment]
Please update this 2 files as well:

Configuration and Loading
Add the new language to Appwrite config file: /app/config/locale/codes.php (order by ABC)
Load the new translation here: ./app/init.php (line: 224) (order by ABC)

File: app/config/locale/ur.countries.php (hunk 1 above)
Comment (navneetsn18) [reply app/config/locale/ur.countries.php:39]: From where to where..

Comment (spielers) [reply app/config/locale/ur.countries.php:39]: 'CD' => 'DR کانگو',

Comment (navneetsn18) [reply app/config/locale/ur.countries.php:39]: Ok i will fix that

**Comment by navneetsn18** [comment]
Ok sure @eldadfux

**Comment by navneetsn18** [comment]
@spielers @eldadfux I have made all the changes required please have a look.

**Review by spielers — APPROVED** [review]
Hey @navneetsn18 

Great job will get you merged !!

Bets,
spielers

**Comment by navneetsn18** [comment]
Thank you @spielers

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**